docs/ROADMAP.md

```diff
@@ -14,10 +14,18 @@ _Setup_ - Completed - 2023/04/20
 1. Migrate files from the
    [CloudEvent repo](https://github.com/cloudevents/spec).
 
-_x.y_ - WIP - 2023/MM/DD
+_Release Candidate 1_ - WIP - 2025/02/28
 
-1. xxx
+* [Core
+  features](https://github.com/xregistry/spec/issues?q=is%3Aissue%20state%3Aopen%20label%3Av1)
+  completed
+* Ready for implementation testing and thorough review of the spec
 
-_Future_
+_Release Candidate 2_ – WIP – 2025/04/30
 
-- TBD.
+* Tested with at least one implementation
```

Comment:
Who would be the implementing party? Inside the group or outside?

Reply:
At some point we should list them - point to our reference impl and mention which companies are implementing it. But we can add that later.